CRM/Member/Form/Membership.php

```diff
@@ -581,7 +581,7 @@ public function buildQuickForm() {
     );
 
     if (!empty($this->_recurPaymentProcessors)) {
-      $memTypeJs['onChange'] = "" . $memTypeJs['onChange'] . "buildAutoRenew(this.value, null, '{$this->_mode}');";
+      $memTypeJs['onChange'] = "" . $memTypeJs['onChange'] . " buildAutoRenew(this.value, null, '{$this->_mode}');";
```

Comment:
Without the space I'm getting a console error when editing a membership that 'buildAutoRenew' is not defined.